.claude/agents/documentation.md

```
@@ -293,19 +292,19 @@ vpc-diagram generate --vpc-id vpc-12345 --output network.svg
 ### 1. Mermaid Diagrams
 **Always use MCP for current syntax:**
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)